scripts/getsamplesummary.py

```diff
@@ -1,90 +1 @@
-
-def getsampleinfofromname(pars, sample):
-  query = (" SELECT sample.sample_id AS id, samplename, flowcellname AS fc, " + 
-           " lane, ROUND(readcounts/2000000,2) AS M_reads, " +
-           " ROUND(q30_bases_pct,2) AS q30, ROUND(mean_quality_score,2) AS score " + 
-           " FROM sample, unaligned, flowcell " + 
-           " WHERE sample.sample_id = unaligned.sample_id AND unaligned.flowcell_id = flowcell.flowcell_id " +
-           " AND (samplename LIKE '" + sample + "_%' OR samplename = '" + sample + "')")
-  with db.create_tunnel(pars['TUNNELCMD']):
-    with db.dbconnect(pars['CLINICALDBHOST'], pars['CLINICALDBPORT'], pars['STATSDB'], 
-                   pars['CLINICALDBUSER'], pars['CLINICALDBPASSWD']) as dbc:
-      replies = dbc.generalquery( query )
-  return replies
-
-def makelinks(samplename, lanedict):
-  if os.path.exists(outputdir + samplename):
-    print outputdir + samplename + ' exists, has data already been exported?'
-    return
-  else:
-    os.makedirs(outputdir + samplename)
-    for entry in lanedict:
-      fclane = lanedict[entry].split("_")
-      print fclane
-      fastqfiles = glob.glob(params['DEMUXDIR'] + "*" + fclane[0] + "*/Unaligned/Project_*/Sample_*" + 
-                            samplename + "_*/*L00" + fclane[2] + "*gz")
-      for fastqfile in fastqfiles:
-        nameparts = fastqfile.split("/")[len(fastqfile.split("/"))-1].split("_")
-        date_fc = fastqfile.split("/")[6].split("_")[0] + "_" + fastqfile.split("/")[6][-9:]
-        newname = (nameparts[3][-1:] + "_" + date_fc + "_" + samplename + "_" + nameparts[2] +
-                   "_" + nameparts[4][-1:] + ".fastq.gz")
-        print fastqfile
-        print newname
-        os.symlink(fastqfile, outputdir + samplename + "/" + newname)
-
-smpls = getsamplesfromflowcell(params, fc)
-
-for sample in smpls.iterkeys():
-  with lims.limsconnect(params['apiuser'], params['apipass'], params['baseuri']) as lmc:
-    analysistype = lmc.getattribute('samples', sample, "Sequencing Analysis")
-    readcounts = .75 * float(analysistype[-3:])    # Accepted readcount is 75% of ordered million reads
-  dbinfo = getsampleinfofromname(params, sample)
-  rc = 0         # counter for total readcount of sample
-  fclanes = {}   # dict to keep flowcell names and lanes for a sample
-  cnt = 0        # counter used in the dict to keep folwcell/lane count
-  for info in dbinfo:
-    if (info['q30'] > 80):     # Use readcount from lane only if it satisfies QC [=80%]
-      cnt += 1
-      rc += info['M_reads']    
-      fclanes[cnt] = info['fc'] + "_" + str(info['q30']) + "_" + str(info['lane'])
-  if (rc > readcounts):        # If enough reads are obtained do
-    print sample + " Passed " + str(rc) + " M reads\nUsing reads from " + str(fclanes)
-    makelinks(sample, fclanes)
-
-  else:                        # Otherwise just present the data
-    print sample + " Fail " + str(rc) + " M reads\nThese flowcells summarixed " + str(fclanes)
```

Comment:
We might as well use the print function since it's the future and includes more functionality. The string concatenation can also be refactored to use a nice mini template syntax:

```python
from __future__ import print_function

print("{sample} Fail {rc} M reads".format(sample=sample, rc=rc))
print("These flowcells summarixed {}".format(fclanes))
```